runtests.sh

```diff
@@ -3,13 +3,19 @@
 # abort immediately on any failure
 set -e
 
+PYTHON_VERSION="${PYTHON_VERSION:-3}"
+
 # if we are not inside the docker container, run this command *inside* the
 # docker container
 if [ ! -f /.dockerenv ]; then
-    docker run -t -i -v $PWD:/python_moztelemetry moztelemetry_docker ./runtests.sh "$@"
+    docker run -t -i -v $PWD:/python_moztelemetry -e PYTHON_VERSION="$PYTHON_VERSION" moztelemetry_docker ./runtests.sh "$@"
     exit $?
 fi
 
+if [ "$PYTHON_VERSION" == '2.7' ]; then
+    export PATH="$PYTHON27_PATH"
+fi
+
```

Comment:
I'm wondering whether we should allow testing against different Python 3 versions as well instead of hardcoding 2.7 and 3.5.

My experience with Conda is limited so I'm not sure if we're depending on the system Python version of the openjdk Docker image (which I think is Debian stretch). Can we try to add 3.6 to the list of Python's and see if it creates a conda env successfully with that?